src/main/java/namor/template/calculator/ClassicCalculator.java

```diff
@@ -0,0 +1,70 @@
+package namor.template.calculator;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+
+public class ClassicCalculator {
+
+    private LinkedList<String> operands;
```

Comment:
It's not good practice to have mutable things which constantly change there conditions as a field. As it uses by few methods it can cause in big tasks loose of data or it's mutation when it didn't have to be changed etc. that's why delete it and make collection as a variable in method.